lab_1_classify_by_unigrams/main.py

```diff
@@ -7,6 +7,14 @@
 
 
 def tokenize(text: str) -> list[str] | None:
+    if isinstance(text, str):
+        text = text.lower()
+        tokens = []
+        for i in text:
```

Comment:
rename the variable, otherwise it is not clear what it contains

Reply:
done